README.md

```diff
@@ -153,7 +153,10 @@ createQemu JSON Sample:
   "os": "l26",
   "cores": 2,
   "sockets": 1,
-  "iso": "local:iso/ubuntu-14.04.5-server-amd64.iso",
+  "iso": {
+    "storage": "local",
+    "file": "ubuntu-14.04.5-server-amd64.iso"
+  },
```

Comment:
I've update this JSON sample because with the previous structure I've got an error during the Unmarshal:
```
./proxmox-api-go installQemu proxmox-node-name < qemu1.json
2024/03/10 11:41:02 json: cannot unmarshal string into Go struct field ConfigQemu.iso of type proxmox.IsoFile
```